setools/infoflow.py

```diff
@@ -344,6 +348,34 @@ def _build_subgraph(self):
 
             self.subG.remove_edges_from(delete_list)
 
+        if self.default_tunable:
+            delete_list = []
+            for s, t in self.subG.edges():
+                edge = Edge(self.subG, s, t)
+                rule_list = []
+                for rule in iter(edge.rules):
+                    try:
+                        if rule.conditional:
+                            bool_enabled = rule.conditional.evaluate()
+                            # if conditional is true then delete the false rules
+                            if bool_enabled:
+                                for false_rule in rule.conditional.false_rules():
+                                    rule_list.append(false_rule)
+                            # if conditional is false then delete the true rules
+                            else:
+                                for true_rule in rule.conditional.true_rules():
+                                    rule_list.append(true_rule)
+                    except setools.exception.RuleNotConditional as e:
```

Comment:
this should be imported like `from .exception import RuleNotConditional`, then you can directly use it.